package.json

```diff
@@ -43,7 +43,8 @@
     "onLanguage:arm-template",
     "onLanguage:json",
```

Comment:
Should we put an editor context menu item to reload the schemas like in VS?  Just add this to the package.json:

        "menus": {
            "editor/context": [{
                "command": "azurerm-vscode-tools.reloadSchemas",
                "when": "editorLangId==arm-template"
            }]
        },

Reply:
Sounds good to me.